docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3"
```

Comment:
Is there any reason of this change?

Reply:
Two out of 4 machines have this (lower) version, so this file will work for everyone